primes.rb

```diff
@@ -1,6 +1,13 @@
 class Primes
   def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+    require 'prime'
+    
+    total = 0
+	
+	Prime.each(limit) { |x| total += x }
```

Comment:
This works, but it doesn't necessarily show your Ruby skills or problem-solving ability. Try taking a look at an algorithm like this and whether you can make sense of it: https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes